spec/features/teacher_auth_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'irb'
```

Comment:
Could be nice to have a line of space between requires and first `describe`. Just aesthetic reasons.

Reply:
also - what `require 'irb'` is here for? 🤔

Reply:
@mtrojanowska - need your input here :)